<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Generatr</class>
 <widget class="QWidget" name="Generatr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>681</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>General Configurations</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>571</width>
       <height>81</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Type</string>
     </property>
     <widget class="QCheckBox" name="check_binary">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Binary</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_even">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Even</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_tri">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_henon">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Henon</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_10dbq1">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>10 dB q =1</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_ternary">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Ternary</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_10dbq2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>10 dB q = 2</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_15dbq1">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>15 dB q = 1</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>571</width>
       <height>81</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>D-Markov</string>
     </property>
     <widget class="QCheckBox" name="check_dmarkov">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_d_ini">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_d_fin">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>66</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>From D =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>66</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>to D=</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>571</width>
       <height>81</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Mk1</string>
     </property>
     <widget class="QCheckBox" name="check_mk1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_mk1_old">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Expansion 1</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_mk1_new">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Expansion 2</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_mk1_dmark">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>D-Markov Expansion</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>571</width>
       <height>81</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Mk2</string>
     </property>
     <widget class="QCheckBox" name="check_mk2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_mk2_old">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Expansion 1</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_mk2_new">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Expansion 2</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_mk2_dmark">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>D-Markov Expansion</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>410</y>
       <width>571</width>
       <height>81</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>L</string>
     </property>
     <widget class="QCheckBox" name="check_multL">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Multiple L</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_L_ini">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_L_fin">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>66</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>From L =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>66</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>to L=</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>66</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Single L =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_single_L">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>500</y>
       <width>571</width>
       <height>81</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Alpha</string>
     </property>
     <widget class="QCheckBox" name="check_multi_alpha">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Multiple Alpha</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_alpha_ini">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_alpha_fin">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>60</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>From alpha =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>66</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>to alpha=</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Single alpha =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_single_alpha">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="saveconf">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>620</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Config</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Generate Graphs</string>
    </attribute>
    <widget class="QPushButton" name="gengraph">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>300</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate Graphs</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="bar_gengraph">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>571</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Generate Sequences</string>
    </attribute>
    <widget class="QPushButton" name="genseq">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>290</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate Sequences</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_original">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate From Original</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>180</y>
       <width>66</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Length:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_length">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>180</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QProgressBar" name="bar_genseq">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>571</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Analyze Sequences</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>571</width>
       <height>81</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Conditional Entropies</string>
     </property>
     <widget class="QCheckBox" name="check_entropy">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_entropy">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>66</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Up to:</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>571</width>
       <height>81</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Autocorrelations</string>
     </property>
     <widget class="QCheckBox" name="check_autocorr">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_autocorr">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>66</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Up to:</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>571</width>
       <height>81</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Kullback-Leibler Divergence</string>
     </property>
     <widget class="QCheckBox" name="check_kld">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_kld">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>66</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Length:</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="analyze">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>440</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Analyze</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="bar_an_seq">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>520</y>
       <width>571</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Plots</string>
    </attribute>
    <widget class="QPushButton" name="plot">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>440</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_25">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>571</width>
       <height>81</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Kullback-Leibler Divergence</string>
     </property>
     <widget class="QCheckBox" name="plot_kld">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_26">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>571</width>
       <height>81</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Autocorrelations</string>
     </property>
     <widget class="QCheckBox" name="plot_autocorr">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_27">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>571</width>
       <height>81</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Conditional Entropies</string>
     </property>
     <widget class="QCheckBox" name="plot_entropy">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
